elastio-stream-kafka/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
+
+# Version elastio cli what will use like base image.
+# Most likely this is not the latest version of elastio cli.
+# You can find the latest version at this link.
+# https://gallery.ecr.aws/elastio-dev/elastio-cli
+FROM public.ecr.aws/elastio-dev/elastio-cli:0.17.0-master
+
+RUN apt-get update && apt-get upgrade -y
```

Comment:
I would not upgrade, rather just taking a needed base image with newest relevant tag.